packaging/debian/prerm

```diff
@@ -21,7 +21,7 @@ case "$1" in
     remove|upgrade)
 	for binary in "storagenode" "storagenode-updater"; do
 	    rm -f /usr/bin/${binary}
-	    rm -f /var/storagenode/${binary}
+	    rm -f /var/lib/storagenode/${binary}
```

Comment:
This is a fairly minor thing, but these `rm`s should be done in postrm, not in prerm. When the package is being removed and `prerm` is run, the service might still be running and expected to work. By the time we get to `postrm`, though, it's safe to clean up extra things like this.